week1/project/ecommerce/src/CategoryList.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prop-types */
+
+
+const CategoryList = ({ categories, selectedCategory, handleCategoryClick }) => {
+    return (
+        <div className="categories">
+        {categories.map((category) => (
+            <div key={category} 
+            className = {`category ${category === selectedCategory ? 'category-selected' : ''}`}>
```

Comment:
⚒️ 

Nice work with the conditional CSS! One thing to mention: there should not be spaces between the attribute and the value.

```suggestion
            className={`category ${category === selectedCategory ? 'category-selected' : ''}`}>
```